### PR TITLE
Create joint validation for attendees 

### PR DESCRIPTION
**Pull request by lauraGgit**
create a joint validator on the two number of attendee (participants and spectators) to let them know that they don't actually need a permit.

### CODE REVIEW

**Comment by hursey013** [comment]
screen shot 2018-08-03 at 1 58 37 pm

File: frontend/src/app/application-forms/fields/noncommercial-fields.component.spec.ts [hunk 1]
```diff
@@ -34,27 +34,33 @@ describe('noncommercial fields', () => {
...
+  it('should be invalid if paticipants is greater than 0 and spectators is greater than 0 but a sum of less than 75', () => {
     component.noncommercialFields.controls['numberParticipants'].setValue(1);
-    expect(component.noncommercialFields.controls['numberParticipants'].valid).toBeTruthy();
+    component.noncommercialFields.controls['numberSpectators'].setValue(1);
+    component.noncommercialFields.controls['activityDescription'].setValue('test');
+    component.noncommercialFields.controls['locationDescription'].setValue('test');
+    expect(component.noncommercialFields.valid).toBeFalsy();
   });
 
-  it('should be invalid if participants is less than 1', () => {
```
Comment (lauraGgit) [inline frontend/src/app/application-forms/fields/noncommercial-fields.component.spec.ts]: we do want to keep this test - we should probably just make it a test of plus for attendees and negative for participants.

File: frontend/src/app/application-forms/fields/noncommercial-fields.component.spec.ts [hunk 2]
```diff
@@ -34,27 +34,33 @@ describe('noncommercial fields', () => {
...
-    expect(component.noncommercialFields.controls['numberParticipants'].valid).toBeFalsy();
+  it('should be valid if paticipants is greater than 0 and spectators is greater than 0 with a sum greater than 75', () => {
+    component.noncommercialFields.controls['numberParticipants'].setValue(30);
+    component.noncommercialFields.controls['numberSpectators'].setValue(45);
+    component.noncommercialFields.controls['activityDescription'].setValue('test');
+    component.noncommercialFields.controls['locationDescription'].setValue('test');
+    expect(component.noncommercialFields.valid).toBeTruthy();
   });
 
-  it('should be invalid if paticipants is 0 and spectators is greater than 0', () => {
```
Comment (lauraGgit) [inline frontend/src/app/application-forms/fields/noncommercial-fields.component.spec.ts]: Can you say why you removed this test? I think the assertion would still be the case.

**Comment by hursey013** [comment]
@lauraGgit yeah those tests should still be there, still working it out locally, was a little too quick to push things up 😬

File: frontend/src/app/application-forms/fields/noncommercial-fields.component.spec.ts [hunk 3]
```diff
@@ -52,11 +52,35 @@ describe('noncommercial fields', () => {
...
+
+  it('should be valid if paticipants is greater than 75 and spectators is greater than 0', () => {
+    component.noncommercialFields.controls['numberParticipants'].setValue(75);
+    component.noncommercialFields.controls['numberSpectators'].setValue(1);
+    component.noncommercialFields.controls['activityDescription'].setValue('test');
+    component.noncommercialFields.controls['locationDescription'].setValue('test');
+    expect(component.noncommercialFields.valid).toBeTruthy();
+  });
+
+  it('should be invalid if paticipants is greater than 0 and spectators is greater than 75', () => {
```
Comment (lauraGgit) [inline frontend/src/app/application-forms/fields/noncommercial-fields.component.spec.ts]: this looks like it should say valid.

**Pull request merged**